integration/env_build.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+#
+#  Copyright (c) 2015, 2016, 2017, 2018, 2019, 2020, Intel Corporation
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions
+#  are met:
+#
+#      * Redistributions of source code must retain the above copyright
+#        notice, this list of conditions and the following disclaimer.
+#
+#      * Redistributions in binary form must reproduce the above copyright
+#        notice, this list of conditions and the following disclaimer in
+#        the documentation and/or other materials provided with the
+#        distribution.
+#
+#      * Neither the name of Intel Corporation nor the names of its
+#        contributors may be used to endorse or promote products derived
+#        from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+#  "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+#  LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+#  A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+#  OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+#  SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+#  LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+#  DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+#  THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+#  (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY LOG OF THE USE
+#  OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+#
+
+# Default GEOPM setup
+GEOPM_SRC=${GEOPM_SRC:?Please set GEOPM_SRC in your environment.}
+GEOPM_BUILD=${GEOPM_BUILD:?Please set GEOPM_BUILD in your environment.}
+GEOPM_APPS_SRCDIR=${GEOPM_APPS_SRCDIR:?Please set GEOPM_APPS_SRCDIR in your environment.}
+
+# Default Intel Toolchain compiler overrides
+CC=icc
+CXX=icpc
+MPICC=mpicc
+MPICXX=mpic++
+FC=ifort
+F77=ifort
+MPIFC=mpifort
+MPIF77=mpifort
+
+# Exports
+echo "Exporting:"
+echo "  GEOPM_SRC    = ${GEOPM_SRC}"
+echo "  GEOPM_BUILD  = ${GEOPM_BUILD}"
+echo "  JOB_WORK_DIR = ${JOB_WORK_DIR}"
+export GEOPM_SRC GEOPM_BUILD JOB_WORK_DIR
+
+echo "  CC     = ${CC}"
+echo "  CXX    = ${CXX}"
+echo "  MPICC  = ${MPICC}"
+echo "  MPICXX = ${MPICXX}"
+echo "  FC     = ${FC}"
+echo "  F77    = ${F77}"
+echo "  MPIFC  = ${MPIFC}"
+echo "  MPIF77 = ${MPIF77}"
+export CC CXX MPICC MPICXX FC F77 MPIFC MPIF77
+
```

Comment:
I'd prefer to keep some of what's below.  I'm thinking of situations where someone is bringing up a new app, and would source this env file in their current shell and expect the env to be ready to start compiling the new app.

For that use case, things like the man pages, Python bindings to GEOPM (if an app used them), and basic checks on the sanity of your GEOPM build (which is required for some apps, e.g. Nekbone since it has markup) seem appropriate to avoid headaches later.